encoding_header.py

```diff
@@ -0,0 +1,36 @@
+"""
+<Author>
```

Comment:
Including the author and start date in the header block is fine in my opinion, although not required.  Albert has stated that a lot of this information is provided by the VCS/Git

The latest format of docstrings:
https://github.com/secure-systems-lab/code-style-guidelines#documentation-strings